help/en/releasenotes/current-draft-warnings.shtml

```diff
@@ -25,6 +27,9 @@
             <li></li>
             <li></li>
             <li></li>
+            <li></li>FileChooserUtil
+            <li></li>get/setFontFamily in SystemConsolePreferencesManager
```

Comment:
Is it intended that the text is after the `</li>` tag?

Reply:
Thanks. Fixed.